src/lib/loaders/loaders_with_authentication/gravity.js

```diff
@@ -41,10 +65,26 @@ export default (accessToken, userID, requestIDs) => {
     ),
     saleArtworksLoader: gravityLoader(id => `sale/${id}/sale_artworks`),
     saleArtworksFilterLoader: gravityLoader("filter/sale_artworks"),
-    suggestedArtistsLoader: gravityLoader("me/suggested/artists", {}, { headers: true }),
-    updateCollectorProfileLoader: gravityLoader("me/collector_profile", {}, { method: "PUT" }),
+    suggestedArtistsLoader: gravityLoader(
+      "me/suggested/artists",
+      {},
+      { headers: true }
+    ),
+    updateCollectorProfileLoader: gravityLoader(
+      "me/collector_profile",
+      {},
+      { method: "PUT" }
+    ),
```

Comment:
Another example of what I mean from above. Could be something like 
```javascript
updateCollectorProfileLoader: gravityLoader("me/collector_profile", {
  apiOptions: {
    method: "PUT"
  }
})
```
Which would clear up some ambiguity and make the code look nicer.